### ISSUE TITLE
[BUG] player.setSpeed(2) is having no effect, speed is still 1 in video.

### ISSUE THREAD

**Issue by Gh05ts** [befovy/fijkplayer#301]
**Describe the bug**
Not able to setSpeed to be greater than 1.

**What attempts have you made to resolve this error ?**
I removed all setOption from my code and tried to run the player, still did not fix.

**To Reproduce**
1. Load any video file
2. Use setSpeed() to change speed
3. No change in speed if speed > 1

**Expected behavior**
setSpeed() should change the speed to the amount set.

**fijkplayer version**
Latest   fijkplayer: ^0.8.7

**Smartphone (please complete the following information):**
OS: Android 10, Phone: Motorola P30

**Additional context**
I encountered the same issue with flutter ijk player (not fijk) in that player if we used setOptions(), the ability to play faster than 1 was also going, but if I did not use setOption() it would play faster than 1. In this version I can't play speeds greater than 1 even if I don't use setOption anywhere. The reason I had to switch from flutter ijk to fijk is because that player is not able to play ipv6 videos, while this player can play ipv6, it is not able to change speeds, pls help.

**Comment by befovy** [comment]
Dose your video file has audio stream？

**Comment by Gh05ts** [comment]
Yes, it's a vimeo hls video, it comes with a audio stream and video playlists

This is what the master file looks like, I've removed the links.
```
#EXTM3U
#EXT-X-INDEPENDENT-SEGMENTS
#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio-medium",NAME="audio",AUTOSELECT=YES,DEFAULT=YES,CHANNELS="1",URI="path/to/file"
#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio-high",NAME="audio",AUTOSELECT=YES,DEFAULT=YES,CHANNELS="1",URI="path/to/file"

#EXT-X-STREAM-INF:CLOSED-CAPTIONS=NONE,BANDWIDTH=249902,AVERAGE-BANDWIDTH=135000,RESOLUTION=426x240,FRAME-RATE=25.000,CODECS="avc1.640015,mp4a.40.2",AUDIO="audio-medium"
path/to/file
#EXT-X-STREAM-INF:CLOSED-CAPTIONS=NONE,BANDWIDTH=383947,AVERAGE-BANDWIDTH=172000,RESOLUTION=640x360,FRAME-RATE=25.000,CODECS="avc1.64001E,mp4a.40.2",AUDIO="audio-medium"
path/to/file
```
